src/airflow_declarative/schema.py

```diff
@@ -120,10 +122,13 @@ def represent_callable(self, obj):
 CLASS = Class()
 DATE = Date()
 EMAIL = Email
+NULL = Null()
 POSITIVE_INT = Int(gte=0)
 STRING = String()
 TIMEDELTA = TimeDelta()
 
+AIRFLOW_NULL = NULL | Regexp(regexp="None")
+CRON_PRESETS = Regexp(regexp=r"@\w+")
```

Comment:
This actually catch a lot of things, while Airflow supports a [limited set](http://airflow.apache.org/scheduler.html#dag-runs) of those. Let's not allow our users to make a mistake.

Reply:
I guess Airflow's DAG should raise an error on `__init__` when an unsupported cron preset is passed. But I wouldn't rely on that either, given that they might decide to move out that validation from `__init__` like they did with dag cycles before.

So I think a better way is to just enumerate them all, maybe with trafaret's `Enum` atom. If something will be missing here (and present in Airflow), this wouldn't be a big deal -- expanding API is easier than shrinking. We already enumerate a bunch of Airflow attributes anyways.

Reply:
I limit presets to listed in documentation with Enum.